src/store/store.ts

```diff
@@ -72,22 +87,32 @@ export const onMouseMoveAtom = atom(
         const newY = selectedElement.y + (update.y - selectingArea.startY)
         set(updateElementsAtom, { ...selectedElement, x: newX, y: newY })
       }
+      if (get(isDrawingAtom) && selectedElement?.x && selectedElement.y) {
```

Comment:
why the first one here is with null check `selectedElement?.x`, but the second one is without `selectedElement.y`?